flake.nix

```
@@ -1,12 +1,9 @@
 {
   description = "ghc-build-analzyer: utility for analyzing GHC build";
   inputs = {
-    nixpkgs = {
-      type = "github";
-      owner = "avieth";
-      repo = "nixpkgs";
-      rev = "346134b9b036a6f5c3f7239b450343c88954acea";
-    };
+    # nixpkgs/master on 2022-10-21
+    nixpkgs.url =
+      "github:NixOS/nixpkgs/71c5816834f93840dd301ec384c9d7947e97c27d";
```

```suggestion
    nixpkgs.url = "github:NixOS/nixpkgs/master";
```
You could have this reference the `master` branch directly, and then the actual commit hash would be stored in `flake.lock`, and you could update it automatically with `nix flake update`.

---

thanks! yes. I will do that. :-)